pages/common/gnmic-get.md

```diff
@@ -0,0 +1,20 @@
+# gnmic get
+
+> Get a snapshot of a gnmi network device operation data.
+> More information: <https://gnmic.kmrd.dev/cmd/get>.
+
+- Get a snapshot of the device state at {{path}}:
+
+`gnmic -a {{ip:port}} get --path {{path}}`
```

Comment:
```suggestion
`gnmic --address {{ip:port}} get --path {{path}}`
```

Long options are preferred. But now there are no fixed rules when we use them. So you for instance can tell me that this option is frequently used or i extremely portable and I dismiss this comment.

Reply:
I have updated the first mentions of `-a` in every page/subpage to follow the long format for clarity but gnmic commands tend to become very long very fast and the address is always included, so I think that using the -a in every other spot is a better choice (once to explain the parameter, the rest is implied)

Reply:
I think we should formalize when to prefer short options not to write very long commands.